main.go

```diff
@@ -26,7 +27,13 @@ var githubClient *github.Client
 func init() {
 	accessToken, loaded := os.LookupEnv("ACCESS_TOKEN")
 	if !loaded {
-		githubClient = github.NewClient(nil)
+		githubToken, _ := os.LookupEnv("GITHUB_TOKEN")
+		ctx := context.Background()
+		ts := oauth2.StaticTokenSource(
+			&oauth2.Token{AccessToken: githubToken},
+		)
+		tc := oauth2.NewClient(ctx, ts)
+		githubClient = github.NewClient(tc)
 		return
 	}
 	transport := &github.BasicAuthTransport{
```

Comment:
I'm not sure if my modification is proper. I may misunderstand your code. Please review them.